extensions/github-authentication/src/common/keychain.ts

```diff
@@ -76,6 +76,11 @@ export class Keychain {
 			return Promise.resolve(undefined);
 		}
 	}
+	onDidChangePassword(listener: () => void) {
```

Comment:
Not clear to me why this indirection is needed and `vscode.authentication.onDidChangePassword` cannot be used directly.